charts/gardener/gardenlet/templates/_helpers.tpl

```diff
@@ -198,6 +198,9 @@ config.yaml: |
         threshold: {{ .Values.config.controllers.shootCare.staleExtensionHealthChecks.threshold }}
         {{- end }}
       {{- end }}
+      {{- if .Values.config.controllers.shootCare.managedResourceProgressingThreshold }}
+      managedResourceProgressingThreshold: {{ .Values.config.controllers.shootCare.managedResourceProgressingThreshold }}
```

Comment:
> you can directly enable it by default.

Are you suggesting, adding defaulting for this value here https://github.com/gardener/gardener/blob/master/pkg/gardenlet/apis/config/v1alpha1/defaults.go?

Reply:
No.

If you add a defaulting, then the field will be always non-nil. Right now when managedResourceProgressingThreshold=nil, we assume that the check is disabled. If we add such defaulting, there will be now way to disable the check, right?

By "you can directly enable it by default." I mean to add to the values.yaml file the value uncommented. 

With the current revision of the PR, the managedResourceProgressingThreshold is not present in values.yaml. We have 2 options:
1. Add it commented as suggested in https://github.com/gardener/gardener/pull/7241#discussion_r1080953038 and enable it in controlled manner on our landscapes. When we gain experience with it, add the managedResourceProgressingThreshold field to the values file without the comment and let the check be enabled by default.
2. Directly add the managedResourceProgressingThreshold field to the values file and let the check be enabled by default.

Reply:
I am only worried about the correct timeout. Is one hour right or wrong? May be we can set default to 1hr and then on landscapes it can be modified. WDYT?